src/app/component/ProductCard/ProductCard.component.js

```diff
@@ -263,16 +290,18 @@ export class ProductCard extends PureComponent {
         return (
             this.renderCardLinkWrapper((
                 <>
-                    <figure block="ProductCard" elem="Figure">
+                    <div block="ProductCard" elem="Figure-Review">
+                        <figure block="ProductCard" elem="Figure">
                         { this.renderPicture() }
```

Comment:
Spacing! The ESLint plugin shouldn't have let this through, is it enabled in your code editor?

Reply:
Yes, I have enabled ESLint. Even linter on github passed, idk why :D